features/step_definitions/tax_relief_warning_steps.rb

```diff
@@ -0,0 +1,32 @@
+Given(/^that I am on the WPCC homepage in my own "([^"]*)"$/) do |language|
+  locale = language_to_locale(language)
+
+  your_details_page.load(locale: locale)
+end
+
+When(/^I enter my personal details$/) do
+  your_details_page.age.set(35)
+  your_details_page.genders.select(
+    I18n.translate('wpcc.details.options.gender.female')
+  )
+  your_details_page.salary_frequencies.select(
+    I18n.translate('wpcc.details.options.salary_frequency.year')
+  )
+  your_details_page.minimum_contribution_button.set(true)
+end
+
+When(/^I enter a "([^"]*)" below the salary band$/) do |salary|
+  your_details_page.salary.set(salary)
+end
+
+When(/^I progress to the results page$/) do
+  step 'I move to your results page'
+end
+
+Then(/^I should see tax relief "([^"]*)"$/) do |warning_message|
+  expect(page).to have_content(warning_message)
```

Comment:
@tomas-stefano the previous expectations set up for anything related to _tax reliefs_ step definitions with arguments. The same steps cannot be repeated with the expectation for the _conditional messages_.

```
3 features/step_definitions/your_results_steps.rb|10 col 77| expect(your_results_page.current_period.tax_relief.text).to eq("(includes tax relief of #{tax_relief})")
4 features/step_definitions/your_results_steps.rb|33 col 24| Then(/^I should see my tax relief for second period as "([^"]*)"$/) do |tax_relief|
5 features/step_definitions/your_results_steps.rb|36 col 76| expect(your_results_page.second_period.tax_relief.text).to eq("(includes tax relief of #{tax_relief})")
6 features/step_definitions/your_results_steps.rb|53 col 24| Then(/^I should see my tax relief for third period as "([^"]*)"$/) do |tax_relief|
7 features/step_definitions/your_results_steps.rb|54 col 75| expect(your_results_page.third_period.tax_relief.text).to eq("(includes tax relief of #{tax_relief})")
```